src/ui/components/search/autocompletecomponent.js

```diff
@@ -188,8 +188,8 @@ export default class AutoCompleteComponent extends Component {
   }
 
   isQueryInputFocused () {
-    return document.activeElement &&
-      document.activeElement.className.includes(this._inputEl.substring(1));
+    return document.activeElement && document.activeElement.getAttribute('class') &&
```

Comment:
just for my edification how does this relate to direct answers? does this end up getting called somehow when you click the thumbs up/down button?

Reply:
Yes! This function is used in setState() for direct answers, and get invoke when the feedback button is click.